main.py

```diff
@@ -16,4 +17,43 @@
     print(item)
 conn.close()
 diff = datetime.datetime.now() - start
-print('End {} seconds '.format(diff.total_seconds()))
+print('End {} seconds '.format(diff.total_seconds()))'''
+
+#Edited Code:
+class GetandSortData:
```

Comment:
Put a class in a different file for better readable. In main.py file, import the class.